### PR TITLE
updating npm packages, including Antd

### PR DESCRIPTION
**Pull request by haraldschilly**
# Description

## [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):
- [ ] Testing instructions are provided, if not obvious
- [ ] Release instructions are provided, if not obvious

### CODE REVIEW

File: src/smc-webapp/r_misc/number-input-backend.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ */
+
+import * as React from "react";
+
+interface Props {
+  number: number;
+  unit?: string;
+}
+
+export const NumberInput: React.FC<Props> = (props: Props) => {
```
Comment (williamstein) [inline src/smc-webapp/r_misc/number-input-backend.tsx:13]: What the heck is this?  It says "Input" but it is clearly not an input.  Confused.

**Comment by williamstein** [comment]
Why is there so much new/modified code here related to r_misc? Are all these changes required by updating antd from 4.7.0 to 4.7.2?  If so, what the hell...?

**Comment by haraldschilly** [comment]
no it's not, the pr isn't done yet

**Comment by williamstein** [comment]
OK, got it.  Just curious.   Maybe a separate PR just upgrading antd?

**Comment by haraldschilly** [comment]
I cleaned this up. newer antd, also another package, and many dependencies (audit fix). I also modernized this "Tip" component, so, for a proper test, hover over a project tab to see that there is still something popping up.

**Pull request merged**